sdk/core/azure-core/src/http/log_policy.cpp

```diff
@@ -37,13 +37,32 @@ inline void AppendHeaders(
   }
 }
 
+inline std::string GetUrlWithoutQuery(Url url)
```

Comment:
Take value by const ref. We should avoid creating copies.
```suggestion
inline std::string GetUrlWithoutQuery(Url const& url)
```